apps/user-ms/src/main/java/it/pagopa/selfcare/user/service/UserServiceImpl.java

```diff
@@ -186,13 +187,30 @@ public Uni<Void> updateUserProductCreatedAt(String institutionId, List<String> u
     }
 
     @Override
-    public Uni<UserResource> getUserById(String userId) {
-        return userRegistryApi.findByIdUsingGET(USERS_WORKS_FIELD_LIST, userId);
+    public Uni<UserDetailResponse> getUserById(String userId, String institutionId, String field) {
```

Comment:
you could name "field" as "fieldToRetrieve" to identify better which is its scope